Fixes doc_749 (#1404)
RedisLabs/redislabs-docs: content/rc/how-to/creating-iam-resources/cloudformation.md

<img alt="Launch RedisCloud template" src="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png"/>
</a>

You can then use the `Outputs` tab to find the data needed to complete the creation of a Cloud Account. For the `accessSecretKey` (i.e. user's access key) and `consolePassword` (user's console password) you'll have to follow the links to the AWS Secrets Manager service, and use that to find the secret values. These values, being secrets, aren't displayed directly by CloudFormation.

If you prefer to use the AWS CLI then the command you need is shown below, substituting your profile where it says `YOUR_PROFILE_HERE` (or using one of the other methods as per the [AWS CLI docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html))

```
export AWS_PROFILE=YOUR_PROFILE_HERE
aws cloudformation create-stack --stack-name RedisCloud --template-url \
https://s3.amazonaws.com/iam-resource-automation-do-not-delete/RedisCloud.yaml \
--capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_NAMED_IAM CAPABILITY_IAM
```

> You can track the status of the cloud formation thus:

Reviewer: ```suggestion
You can track the status of the cloud formation with the following command:
```

```
aws cloudformation describe-stacks --stack-name RedisCloud
```
The data needed to complete the creation of a Cloud Account is shown as `Output Key` and `Output Value` pairs.

For the two secrets (`accessSecretKey` and `consolePassword`) you'll need to use the AWS secretmanager CLI - the value you'll need has a key of `SecretString`:

```
aws secretsmanager get-secret-value --secret-id=/redislabsuser/secret_access_key
```

> We recommend for the `consolePassword` you use yaml output - it makes decoding the required value easier

Reviewer: ```suggestion
We recommend using yaml output for the `consolePassword`, as it makes decoding the required value easier.
```
